### ISSUE TITLE
Widget not working since update by Microsoft

### ISSUE THREAD

**Issue by hypedcolee7881** [michalleptuch/calendar-flyout#50]
Microsoft recently updated the widgets platform, and since then, the Calendar Flyout widget will not work. It's a pity because it was so awesome before!

![Screenshot 2024-10-31 105431](https://github.com/user-attachments/assets/2ca37735-6e9a-448b-8797-1430f166be83)

**Comment by michalleptuch** [comment]
What version of Widgets Board do you have?

**Comment by hypedcolee7881** [comment]
Version: 524.28801.30.0

![Screenshot 2024-10-31 161341](https://github.com/user-attachments/assets/7d01abc1-b29a-4526-a509-459d45d96c81)

**Comment by hypedcolee7881** [comment]
Additionally, after the update by Microsoft, I had to 'fix' all of my synced accounts including exchange accounts and one google account. But after clicking on the Fix button, they all work fine. (except for the widget of course).

**Comment by michalleptuch** [comment]
I reported the problem in official Microsoft repo: https://github.com/microsoft/WindowsAppSDK/issues/4841

**Comment by hypedcolee7881** [comment]
thank you, great job!

**Comment by hypedcolee7881** [comment]
Interesting. The version number of my widgets board now reads 524.28801.70.0, so this should be higher, but I got back the old widgets platform with its older look and menu options; and in this older version, the Calendar Flyout widget works fine.

**Comment by michalleptuch** [comment]
Yeah, I think they can control the Widgets Board settings with silent updates. This new board is broken and shouldn't have been released to production. Anyway, I'll keep this issue open to monitor the bug.

**Comment by hypedcolee7881** [comment]
I just got back the new widgets board again. Calendar Flyout widget is not working.

**Comment by michalleptuch** [comment]
Will be fixed in the next release
